ShoeBox/Classes/GUI/MapScreen/ViewController/MapViewController.swift

```diff
@@ -9,20 +9,21 @@
 import MapKit
 import UIKit
 
-class MapViewController: UIViewController, MKMapViewDelegate, CLLocationManagerDelegate {
+class MapViewController: UIViewController, MKMapViewDelegate, LocationManagerDelegate {
     
     fileprivate struct MapViewConstants {
         // segue id
         static let suggestionSegueIdentifier = "ShowMapLocationDetailsScreenIdentifier"
     }
     
     @IBOutlet weak var mapView: MKMapView!
```

Comment:
Example: 

private lazy var locationManager: LocationManager = {
   let manager =  LocationManager()
   manager.delegate = self

return manager

}()

and in viewDidLoad() {
locationManager. startUpdatingLocation()
}